src/ray/protobuf/gcs_service.proto

```diff
@@ -175,6 +175,12 @@ message GetAllNodeInfoRequest {}
 message GetAllNodeInfoReply {
   GcsStatus status = 1;
   repeated GcsNodeInfo node_info_list = 2;
+  GcsInfo gcs_info = 3;
```

Comment:
Can we model gcs_info as a node info here?

Reply:
I feel mixing gcs info (e.g., pending task stats) with worker node info (e.g., static node config) within this RPC is weird. So I just created a separate RPC to get gcs stats (`GetGcsStats`), which makes both clean and clear. Let me know if you have any suggestions. @rkooo567 @iycheng

Reply:
I ask this just because we should go to the direction what scheduler doesn't know raylet/gcs. But if this is only for enabling the gcs based actor scheduling, I'm ok with this.